client/state/atomic-transfer/hooks/use-atomic-transfer-query.ts

```diff
@@ -0,0 +1,61 @@
+import { useQuery } from '@tanstack/react-query';
+import wpcom from 'calypso/lib/wp';
+import { transferStates, TransferStates } from 'calypso/state/automated-transfer/constants';
+import { SiteSlug } from 'calypso/types';
+
+interface ErrorResponse {
+	code: 'no_transfer_record';
+}
+
+interface SuccessResponse {
+	status: TransferStates;
+}
+
+const fetchLatestAtomicTransfer = (
+	siteSlug: SiteSlug
+): Promise< ErrorResponse | SuccessResponse > =>
+	wpcom.req.get( {
+		path: `/sites/${ siteSlug }/atomic/transfers/latest`,
+		apiNamespace: 'wpcom/v2',
+	} );
+
+interface UseAtomicTransferQueryOptions {
+	refetchInterval?: number;
+}
+
+const endStates = [
+	transferStates.NONE,
+	transferStates.COMPLETE,
+	transferStates.FAILURE,
+	transferStates.ERROR,
+	transferStates.REVERTED,
```

Comment:
Should this list of end states also include `RELOCATING_REVERT`? I don't know how to produce this state, but [the comment doc](https://github.com/Automattic/wp-calypso/blob/a76fb98cf130dd021d70ce9378fa5f7b75e9b5e4/client/state/automated-transfer/constants.ts#L19) makes it seems like it's the same sort of thing as `NONE` and `REVERTED`